spec/spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 require "bundler/setup"
 require "neo4j/http"
-require "pry"
```

Comment:
Remove this because it is not declared in the gemspec

Reply:
Weird - I could have sworn I added it to the Gemfile - thoughts on adding it there instead of removing it here?
```
gem "pry", group: [:development, :test]
```

I find pry invaluable when debugging

Reply:
Its not the Gemfile of the project it is included into, it is in the .gemspec of the gem itself. I can add this if you want to keep it